CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Release 2.4.2 (in development)
 
+### Breaking changes
```

Comment:
Would probably make sense to bump the version of `BootstrapConfiguration`

Reply:
No. What you can put in the 'object' doesn't really change: there are no backwards-incompatible changes or anything. We just now ignore information we did use before (and as such, no longer require them to be set). It's a required value that now becomes kinda 'optional' (and unused, at all).

Basically, any existing `v1alpha2` `BootstrapConfiguration` *still* works.